From 9c8d862b2b9eff77f39bbcf77c477c13de3c80ec Mon Sep 17 00:00:00 2001
From: Fei Jiang <fei.jiang@intel.com>
Date: Mon, 10 Sep 2018 10:56:15 +0800
Subject: [PATCH 2116/2313] drm/i915/gvt: also assign pipe C scaler ownership
 to UOS 1

Currently we mainly support one SOS and one UOS, there is usr case that
SOS displays on pipe A, UOS displays video with clone mode on pipe B and C,
if pipe C scaler ownership is not assigned to UOS 1, video on pipe C can
only be displayed with source size on the top-left side of screen.

Signed-off-by: Min He <min.he@intel.com>
Signed-off-by: Fei Jiang <fei.jiang@intel.com>
Reviewed-by: Zhao Yakui <yakui.zhao@intel.com>
---
 drivers/gpu/drm/i915/i915_params.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/i915_params.c b/drivers/gpu/drm/i915/i915_params.c
index 81c2918..5d46c69 100644
--- a/drivers/gpu/drm/i915/i915_params.c
+++ b/drivers/gpu/drm/i915/i915_params.c
@@ -86,7 +86,7 @@ struct i915_params i915_modparams __read_mostly = {
 	.domain_plane_owners = 0,
 	.bg_color = 0x00000000,
 	.gvt_emulate_hdmi = true,
-	.domain_scaler_owner = 0x21100,
+	.domain_scaler_owner = 0x11100,
 };
 
 i915_param_named(modeset, int, 0400,
-- 
2.7.4

